lib/template-mixins.js

```diff
@@ -164,20 +192,41 @@ module.exports = function (fields, options) {
                         value = obj.value;
                         label = obj.label;
                         toggle = obj.toggle;
+                        childTemplate = obj.childTemplate;
                     }
 
                     if (this.values && this.values[key] !== undefined) {
                         selected = this.values[key] === value;
                     }
 
+                    function getErrorClass() {
+                        return function(k) {
+                            k = extractKey.call(this, k);
+                            if (this.errors && this.errors[k]) {
+                                return 'validation-error';
+                            }
+                            return '';
+                        };
+                    }
+
                     return {
                         label: t(label) || '',
                         value: value,
                         selected: selected,
-                        toggle: toggle
+                        toggle: toggle,
+                        childTemplate: childTemplate,
+                        getErrorClass: getErrorClass
```

Comment:
This is so I am able to access the error class from a partial located in the downstream application - https://github.com/UKHomeOffice/gro/blob/bugfix/GFM-9/apps/common/views/partials/toggle-panel.html#L1